scripts/west_commands/runners/nrf_common.py

```diff
@@ -425,22 +431,59 @@ def reset_target(self):
     def do_require(self):
         ''' Ensure the tool is installed '''
 
+    def _get_suit_starter(self):
+        suit_starter = Path(os.environ.get('ZEPHYR_BASE')) / SUIT_STARTER_PATH
+        if not suit_starter.exists():
+            raise RuntimeError("Unable to find suit manifest starter file, "
+                               "please make sure to run \'west blobs fetch "
+                               "hal_nordic\'")
```

Comment:
Shouldn't we just warn? This makes us require a blob to flash...

Reply:
Only if you provide `--suit-manifest-starter` in the command-line.